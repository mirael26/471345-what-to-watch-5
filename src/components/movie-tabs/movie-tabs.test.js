import React from "react";
import renderer from "react-test-renderer";

import MovieTabs from "./movie-tabs";

it(`Should MovieTabsrender correctly`, () => {
  const noop = () => {};
  const tree = renderer
    .create(
        <MovieTabs
          viewType={`overview`}
          onTabClick={noop}
        />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
