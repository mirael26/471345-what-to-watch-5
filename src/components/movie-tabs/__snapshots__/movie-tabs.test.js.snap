// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should MovieTabsrender correctly 1`] = `
<nav
  className="movie-nav movie-card__nav"
>
  <ul
    className="movie-nav__list"
    onClick={[Function]}
  >
    <li
      className="movie-nav__item movie-nav__item--active"
    >
      <a
        className="movie-nav__link"
        href="#"
        id="overview"
      >
        Overview
      </a>
    </li>
    <li
      className="movie-nav__item"
    >
      <a
        className="movie-nav__link"
        href="#"
        id="details"
      >
        Details
      </a>
    </li>
    <li
      className="movie-nav__item"
    >
      <a
        className="movie-nav__link"
        href="#"
        id="reviews"
      >
        Reviews
      </a>
    </li>
  </ul>
</nav>
`;
